.travis.yml

```diff
@@ -3,9 +3,6 @@ dist: trusty
 language: ruby
 cache: bundler
 before_install:
-  - if [ $BUNDLER_VERSION ]; then

```

Comment:
Note, this does more than just fix reboot. It removes pupept 4, so we need to start syncing the other modules too